docs/API/handle-api-limit.code.js

```diff
@@ -7,5 +7,5 @@ const Client = api.Client;
 
 const client = new Client({
 	token: "your api token",
-	use_backoff_sleep: false
+	useBackoffSleep: false
```

Comment:
Let's default to true

Reply:
I think it's an example of sentence `This behavior can be disabled and throw exception instead of sleep.` 
https://webscraper.io/documentation/web-scraper-cloud/api

Still change to true? Php example has `false` as well